package.json

```diff
@@ -28,15 +29,23 @@
     "@babel/core": "^7.1.2",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
@gorakong Consider separating your dependencies into dependencies and dev dependencies.